src/components/diamond.jsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+const Angle = '30deg';
```

Comment:
Because it was done like in the block-header component, and all other components with `const` values